atomic_reactor/schemas/config.json

```diff
@@ -280,38 +230,6 @@
         "additionalProperties": false,
         "required": ["api_url", "auth", "signing_intents", "default_signing_intent"]
     },
-    "omps": {
```

Comment:
Should this also go in the Remove OMPS commit?

Reply:
moved there